src/caches/ssprk_caches.jl

```diff
@@ -321,8 +321,8 @@ end
   u::uType
   uprev::uType
   k::rateType
-  tmp::uType
   fsalfirst::rateType
+  tmp::uType
```

Comment:
You can also override https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/src/integrators/integrator_interface.jl#L82 on these methods. `k` might be a suitable cache?

Reply:
But that would work only if `uType === rateType`, wouldn't it? That's definitely okay in our case but won't work if there are units or something similar. However, since the 2N methods already do this and SSP methods are similarly specialized and not really general-purpose methods, that should be okay. I'll do that.

Reply:
Done in 66c83ec.